_config/requestprocessors.yml

```diff
@@ -16,3 +16,10 @@ SilverStripe\Core\Injector\Injector:
   SilverStripe\Control\TrustedProxyMiddleware:
     properties:
       TrustedProxyIPs: "`SS_TRUSTED_PROXY_IPS`"
+---
+Name: errorrequestprocessors
+After:
+  - requestprocessors
+---
+SilverStripe\Core\Injector\Injector:
```

Comment:
It just imports the same dependencies (Middlewares) but doesn't override the class property (director config doesn't specify it).